include/mtrie.h

```diff
@@ -4,19 +4,18 @@
 
```

Comment:
Why does this exist given that `trie.h` exists?

Reply:
Deleted.